collections/_posts/2022-10-17-DiffusionModel.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: review
+title: "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+author: "Sophie Carneiro Esteves"
+cite:
+    authors: "Jascha Sohl-Dickstein, Eric A. Weiss, Niru Maheswaranathan and Surya Ganguli"
+    title:   "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+    venue:   "International Conference on Machine Learning"
+pdf: "http://proceedings.mlr.press/v37/sohl-dickstein15.pdf"
+---
+
+# Notes
+- Code is available on GitHub: [https://github.com/Sohl-Dickstein/
+Diffusion-Probabilistic-Models](https://github.com/Sohl-Dickstein/
+Diffusion-Probabilistic-Models)
```

Comment:
Even if the line in markdown becomes long because of it, I think it's simpler to keep the hyperlink on a single line:
```suggestion
- Code is available on GitHub: [https://github.com/Sohl-Dickstein/Diffusion-Probabilistic-Models](https://github.com/Sohl-Dickstein/Diffusion-Probabilistic-Models)
```